oracle-server-ts/lib/index.ts

```diff
@@ -146,3 +148,25 @@ export function GetSignatures(eventName: string) {
     return <OracleResponse<OracleEvent>>response
   })
 }
+
+export function DeleteAnnouncement(eventName: string) {
+  console.debug('DeleteAnnouncement()')
```

Comment:
might be worth adding the name of the event we are deleting to the `console.debug()`

Reply:
Done